test/suites/integration/byoc_test.go

```diff
@@ -0,0 +1,412 @@
+package integration
```

Comment:
Nice clean logic here. I do wonder if longer term we could look at using [ACK](https://github.com/aws-controllers-k8s/community to simplify management of non-Lattice AWS resources for these tests.

Reply:
Do you think ACK will be easier than SDK? I didnt have issues with SDK so far, well documented, easy to use.

Reply:
Using ACK just means we can model our desired state rather than coding the steps to reach that state. It's a small but perhaps useful difference - we'll really only know if someone tries it out.